src/v/cluster/feature_manager.cc

```diff
@@ -153,6 +153,10 @@ ss::future<> feature_manager::start() {
           [this] { return maybe_log_license_check_info(); });
     });
 
+    for (const model::node_id n : seed_nodes) {
```

Comment:
I think this makes sense on first startup, but not on subsequent upgrades.

When we upgrade, we should not fast-forward all nodes to the latest version: they have to come up and report their version as normal, otherwise we might start sending new-format messages to old-version nodes.

Reply:
`seed_nodes` is only populated when we are starting up a new cluster. It contains either 
* the root node on the root node in the old way of cluster bootstrap (root-driven), or 
* all seed severs on a seed server, and all other seed servers have been verified to be at the same logical version as the local (in seeds-driven bootstrap).

I think I can name this parameter better to reflect that.